charts/castai-evictor/child-charts/castai-evictor-ext/templates/clusterrole.yaml

```diff
@@ -75,12 +75,6 @@ rules:
   # ------------------------------------------------
   # Live Migration: retrieving and creating migrations
   # ------------------------------------------------
-  - apiGroups:
-      - cast.ai
-    resources:
-      - migrations
-    verbs:
-      - get
-      - list
-      - watch
-      - create
\ No newline at end of file
+  - apiGroups: [ "live.cast.ai" ]
+    resources: [ "*" ]
+    verbs: [ "get", "list", "watch", "patch", "create", "delete", "update" ]
```

Comment:
question: why do we need broader permissions that previously?

Is this intended to just catch all possible permissions for the future? Or is there a legitimate need for them now?

In any case, please make sure that Evictor's current check for permissions is up-to-date. I’ll send the relevant code through a different channel.